atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/Value.java

```diff
@@ -56,6 +56,10 @@ public static Value createWithCopyOfData(byte[] contents, long timestamp) {
         return contents;
     }
 
+    public boolean isEmpty() {
```

Comment:
not sure i follow?

Reply:
It's not something you added, but I think `Value.createWithCopyOfData(null, 1L)` will cause an NPE, whereas `Value.create(null, 1L)` doesn't.

I only spotted it because I was figuring out if `isEmpty()` could also cause an NPE (turns out it can't).